QMChatService/QMChatService/Categories/QBChatMessage+QMCustomParameters.h

```diff
@@ -31,6 +31,8 @@
 @property (strong, nonatomic) NSString *dialogPhoto;
 @property (strong, nonatomic) NSDate *dialogUpdatedAt;
 
+@property (strong, nonatomic) NSString *stickerMessage;
```

Comment:
In our terms this string called "Sticker Message", it's named like that everywhere in StickerPipe. We have Sticker ID also which is included in the Sticker Message. Example:" [[contentId]]"